src/FsLexYacc.Runtime/Lexing.fs

```diff
@@ -400,7 +402,13 @@ type UnicodeTables(trans: uint16[] array, accept: uint16[]) =
                     // which covers all Unicode characters not covered in other
                     // ways
                     let baseForUnicodeCategories = numLowUnicodeChars + numSpecificUnicodeChars * 2
-                    let unicodeCategory = System.Globalization.CharUnicodeInfo.GetUnicodeCategory(inp)
+
+                    let unicodeCategory =
+#if FABLE_COMPILER
```

Comment:
I know that these can differ, see Remarks [here](https://learn.microsoft.com/en-us/dotnet/api/system.globalization.charunicodeinfo.getunicodecategory?view=net-8.0)
But in practice, I think it's acceptable.